src/usecase.js

```diff
@@ -1,7 +1,10 @@
 const { step } = require('./step')
 const { Err } = require('./results')
 const { schema } = require('./schema')
+const { v4: uuidv4 } = require('uuid')
```

Comment:
pls, check if you are working on the correct version. a more recent version removed uuid pkg dependency.

Reply:
Fixed!